From cab66eb377f615695fedd7443d0b5d3fdb6cb48d Mon Sep 17 00:00:00 2001
From: saul <saul@crazyauntgail.com>
Date: Wed, 25 Dec 2013 22:56:43 +0000
Subject: Bug 721058 - Shadow layer for Drop Shadow filter in wrong stack position

Fix layer stack positioning for Drop Shadow.

(cherry picked from commit 598673bc830d54c8a88bca3bb85f0565dfc4d011)
---
diff --git a/plug-ins/script-fu/scripts/drop-shadow.scm b/plug-ins/script-fu/scripts/drop-shadow.scm
index 9f6cc2a..d7f8f9e 100644
--- plug-ins/script-fu/scripts/drop-shadow.scm
+++ plug-ins/script-fu/scripts/drop-shadow.scm
@@ -128,6 +128,7 @@
                                             "Drop Shadow"
                                             shadow-opacity
                                             NORMAL-MODE)))
+    (gimp-image-set-active-layer image drawable)
     (gimp-image-insert-layer image shadow-layer 0 -1)
     (gimp-layer-set-offsets shadow-layer
                             shadow-offset-x
--
cgit v0.9.2
